scss/_button-group.scss

```diff
@@ -17,7 +17,8 @@
   > .btn:hover,
   > .btn:focus,
   > .btn:active,
-  > .btn.active {
+  > .btn.active,
+  &:focus-within { // Boosted mod
```

Comment:
For a Boosted mod on multiline, please use:

// Boosted mod:
<..>
// End mod